.changeset/giant-dodos-knock.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/factory': patch
+---
+
+Trimmed what the Factory UI fetches while it polls.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)